### PR TITLE
1.7.6 Release Notes

### PR DESCRIPTION
**Pull request by bhatfield**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,46 +1,45 @@
...
 
----
-### Release 1.7.2 - (07/29/2015)
+##### Themes Enhancement
 
-#### Fixed
-  * [ALOY-1288](https://jira.appcelerator.org/browse/ALOY-1288): Windows: Label not visible when using default app.tss
-  * [ALOY-1287](https://jira.appcelerator.org/browse/ALOY-1287): Alloy styles compilation does not produce the
-    same results across successive compilations
+Themes now support `i18n` and `lib` folders for theme-specific localized assets and CommonJS modules, respectively.
```
Comment (FokkeZB) [inline CHANGELOG.md]: Also `platform`

https://jira.appcelerator.org/browse/ALOY-858

**Comment by FokkeZB** [comment]
We should also mention:

* https://jira.appcelerator.org/browse/ALOY-1237 since it changes what `$.destroy()` does
* https://jira.appcelerator.org/browse/ALOY-967 since it add support for i18n in widgets
* https://jira.appcelerator.org/browse/ALOY-1092 since more advanced querying is often requested
* https://jira.appcelerator.org/browse/ALOY-1173 since it adds copy/move/rename Alloy CLI commands

**Comment by bhatfield** [comment]
Moved content to https://github.com/appcelerator/alloy/releases

**Comment by FokkeZB** [comment]
huh? So you're saying we won't have any changelog doc anymore for Alloy?

**Pull request closed without merging**